pkg/karmadactl/promote/promote.go

```diff
@@ -127,6 +135,9 @@ type CommandPromoteOption struct {
 
 // AddFlags adds flags to the specified FlagSet.
 func (o *CommandPromoteOption) AddFlags(flags *pflag.FlagSet) {
+	flags.BoolVar(&o.AutoCreatePropagationPolicy, "auto-create-propagation-policy", true, "Create a PropagationPolicy automatically")
+	flags.StringVar(&o.PropagationPolicyName, "propagation-policy-name", "",
+		"The name of a PropagationPolicy that automatically created by command. If not specified, a suffix will be added to the PropagationPolicy")
```

Comment:
The flag `auto-create-propagation-policy` might be misleading, it would be better to describe it more clearly.

My suggestion:
```suggestion
	flags.BoolVar(&o.AutoCreatePolicy, "auto-create-policy", true,
		"Automatically create a PropagationPolicy for namespace-scoped resources or create a ClusterPropagationPolicy for cluster-scoped resources.")
	flags.StringVar(&o.PolicyName, "policy-name", "",
		"The name of the PropagationPolicy(or ClusterPropagationPolicy) that is automatically created after promotion. If not specified, the name will be the resource name with a random hash suffix.")
```

Reply:
I think it's really more appropriate that include a description of namespace-scoped resources and cluster-scoped resources.

One thing. @lonelyCZ had mentioned that the default name suffix is not "random" but a specific hash code actually. Do we consider to add "random" word to the description?

Reply:
How about `a specific hash suffix`?

Reply:
> How about `a specific hash suffix`?
If we are on a agreement here I'm gonna make the change.:)

Reply:
`with a hash suffix that is generated by resource metadata`?

Reply:
no big deal. I mean random because from user's respective he/she can't know the hash value and doesn't need to know.

Reply:
Got it.

Reply:
@RainbowMango Change be done. Need one more trigging to CI, and review labels 😄